onboarding-enabler-nodejs-sample-app/package.json

```diff
@@ -13,7 +13,7 @@
   "author": "",
   "license": "EPL-2.0",
   "dependencies": {
-    "apiml-onboarding-enabler-nodejs": "1.0.2",
+    "@zowe/apiml-onboarding-enabler-nodejs": "*",
```

Comment:
Really * ?

Reply:
Ah sorry, it's wrong, I meant latest, I will fix it